src/Models/Hooks/useTimeSeriesData.ts

```diff
@@ -137,16 +138,20 @@ export const useTimeSeriesData = ({
         }
     };
 
+    useEffect(() => {
+        setIsLoading(timeSeriesData.isLoading);
```

Comment:
do we still need local state if it will always refelct the time series data value? Can we just use it directly?

Reply:
since I am exposing isLoading from the useTimeSeriesData hook it needs to be a state variable to let the parent know about a change in the exposed property. Otherwise if I expose isLoading as timeSeriesData.isLoading it would not be triggering consumer of the hook to catch this change.